src/main/python/rlbot/botmanager/helper_process_manager.py

```diff
@@ -7,6 +7,8 @@
 
 from rlbot.botmanager.bot_helper_process import BotHelperProcess
 
+from rlbot.utils.queue import Queue as _queue
+mp.Queue = _queue
```

Comment:
Following previous commit, is this correct?

Reply:
This should follow the same pattern as in setup_manager.py
https://github.com/RLBot/RLBot/pull/499/commits/4600e0fd58a55c37aefe2d5bbd92203308f63ae5#